### PR TITLE
add Lua rockstarness

### PR DESCRIPTION
**Pull request by azet**
Coz Lua Rocks more than any Rockstar developer.

### CODE REVIEW

**Comment by avinassh** [comment]
Surprised we didn't have Lua all these days!

File: examples/lua_rockstar.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from rockstar import RockStar
+
+lua_code = "print 'hello world!'"
+rock_it_bro = RockStar(days=400, file_name='ohai.lua', code=lua_code)
```
Comment (avinassh) [inline examples/lua_rockstar.py]: can change the file name to `helloworld.lua`? 

And also squash the both commits into one?

**Comment by azet** [comment]
@avinassh suggested change amended. HTH.

**Comment by avinassh** [comment]
Thanks!

**Pull request merged**